src/main/java/space/obminyashka/items_exchange/service/impl/UserServiceImpl.java

```diff
@@ -131,11 +131,8 @@ public void update(User user) {
     }
 
     @Override
-    public String updateUserPassword(UserChangePasswordDto userChangePasswordDto, User user) {
-        user.setPassword(bCryptPasswordEncoder.encode(userChangePasswordDto.getNewPassword()));
-        userRepository.saveAndFlush(user);
-
-        return getMessageSource(ResponseMessagesHandler.PositiveMessage.CHANGED_USER_PASSWORD);
+    public String updateUserPassword(String newUserPassword) {
+        return bCryptPasswordEncoder.encode(newUserPassword);
```

Comment:
You should check what you're doing that and how that affects the method's name

Reply:
rename